services/user-services.js

```diff
@@ -0,0 +1,46 @@
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
+const userDao = require('../models/user.dao');
+const {checkEmail}  = require('../validators');
+
+const signUp = async(email, password, firstname, lastname, skintype) => {
+ 
+  await checkEmail(email)
+  const user = await userDao.connectuser(email);
+    if(user){
+    const err = new Error('duplicated email');
+    err.statusCode = 400;
+    throw err;
+  }
+  const hashedPassword = await bcrypt.hash(password, 10);
+  await userDao.createuser(email, hashedPassword, firstname, lastname, skintype); 
```

Comment:
함수가 종료되는 부분에서 return 키워드를 통해 그 이후의 의도치 않은 코드가 실행되지 않도록 수정해주세요
```suggestion
  return await userDao.createuser(email, hashedPassword, firstname, lastname, skintype); 
```